src/constants/numerical.js

```diff
@@ -37,7 +37,7 @@ module.exports = {
     ONEHOUR: 3600000,
     ONEMIN: 60000,
     ONESEC: 1000,
-
+    ONEMICROSEC: 0.001,
```

Comment:
Let's add `ONEMILLI` and use it in your function below i.e. to make it clear.
```suggestion
    ONEMILLI: 1,
    ONEMICROSEC: 0.001,
```

Reply:
good idea, done